apps/camera/js/lib/camera/focus.js

```diff
@@ -81,6 +81,7 @@ Focus.prototype.configureFocusModes = function(mode) {
   this.faceDetection =
     faceDetectionUserEnabled && faceDetectionSupported && mode === 'picture';
   this.mozCamera.addEventListener('focus', this.onAutoFocusStateChange);
+  this.updateFocusState('undefined');
```

Comment:
when camera app is launch for the second time, this.focusState would be equals to 'focusing', so updateFocusState would reset the focusState to the default value.